include/tuvx/linalg/linalg.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <iostream>
+#include <random>
+#include <vector>
+
+namespace tuvx {
+namespace linalg {
```

Comment:
I don't think we've settled on a strategy for when to introduce a new namespace (aside from having one for the library as a whole). I'm fine with introducing one for linear algebra, but am curious about what the rest of the group thinks and whether we should formalize some approach to introducing namespaces?

Reply:
I don't think it's necessary; since we will be using full names `tridiag_solve`, rather than like 3 characters (looking at you lapack), we won't be conflicting with any packages we bring in, probably

Reply:
I have removed the linalg namespace specification